# Include Slackware specific compile flags, provide symlinks for
# backwards compatibility
# Copyright (C) 2012 Georgi D. Sotirov <gdsotirov@dir.bg>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License
# as published by the Free Software Foundation; either version 2
# of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
#
# Visit SlackPack at http://sotirov-bg.net/slackpack/
#
# $Id: qhull-2012.1.diff,v 1.1 2012/03/03 10:26:54 gsotirov Exp $
#

diff -urNad qhull-2012.1-orig/Makefile qhull-2012.1/Makefile
--- qhull-2012.1-orig/Makefile	2012-02-20 04:55:57.000000000 +0200
+++ qhull-2012.1/Makefile	2012-03-03 12:19:53.000000000 +0200
@@ -87,11 +87,11 @@
 
 #for Gnu's gcc compiler, -O2 for optimization, -g for debugging
 CC        = gcc
-CC_OPTS1  = -O2 -fPIC -ansi -Isrc/libqhull $(CC_WARNINGS)
+CC_OPTS1  = -O2 -fPIC -ansi -Isrc/libqhull $(CC_WARNINGS) $(SLKCFLAGS)
 CXX       = g++
 
 # libqhullcpp must be before libqhull
-CXX_OPTS1 = -O2 -Dqh_QHpointer -Isrc/ -Isrc/libqhullcpp -Isrc/libqhull $(CXX_WARNINGS)
+CXX_OPTS1 = -O2 -Dqh_QHpointer -Isrc/ -Isrc/libqhullcpp -Isrc/libqhull $(CXX_WARNINGS) $(SLKCXXFLAGS)
 
 # for shared library link
 CC_OPTS3  =
@@ -191,6 +191,8 @@
 	cp src/libqhull/*.h src/libqhull/*.htm $(INCDIR)/libqhull
 	cp src/libqhullcpp/*.h $(INCDIR)/libqhullcpp
 	cp src/qhulltest/*.h $(INCDIR)/libqhullcpp
+	cd $(INCDIR) && ln -s libqhull qhull
+	cd $(INCDIR)/libqhull && ln -s libqhull.h qhull.h
 
 new:	cleanall all
 
